app/utils/prepareAnswer.js

```diff
@@ -18,6 +18,17 @@ function prepareStatsCompactAnswer(body, index, nameState) {
         data += '<i> (+' + numberWithIndianCommas(statedata['deltaconfirmed']) + ')</i>'
     }
     data += "\n\u{1F7E0} Active: " + numberWithIndianCommas(statedata['active'])
+    if (parseInt(statedata['deltarecovered']) > 0) {
+        let deltaActive = parseInt(statedata['deltaconfirmed']) - parseInt(statedata['deltarecovered']) - parseInt(statedata['deltadeaths'])
+        if (deltaActive > 0)
+            data += '<i> (+' + numberWithIndianCommas(deltaActive) + ')</i>'
+        else if (deltaActive < 0) {
+            deltaActive = deltaActive + (-2) * deltaActive
+            data += '<i> (-' + numberWithIndianCommas(deltaActive) + ')</i>'
+        }
+        else
+            data = data
```

Comment:
We can remove this too

Reply:
yes true i will remove this one as it adds nothing to the data 👍